osf/models/cedar_metadata.py

```diff
@@ -4,13 +4,14 @@
 
 
 class CedarMetadataTemplate(ObjectIDMixin, BaseModel):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, default=None)
```

Comment:
For myself: test if every template has a title, and where does the title come from

Reply:
Yes, `title` seems to be automatically generated from the `name`.